bootstrap_customizer/utils.py

```diff
@@ -51,6 +52,8 @@ def generate_above_the_fold_css(theme):
     """
     context = get_scss_template_context(theme)
     scss_string = render_to_string('bootstrap_customizer/bootstrap_above_the_fold.scss', context)
+    if platform == "win32" or platform == "cygwin":
```

Comment:
@vincent-picot Would we be able to move this to a function? It's repeated a couple times, so it would be DRYer. Something like:

```python
def is_windows_environment():
    return platform == 'win32' or platform == 'cygwin'
```

Reply:
Thanks for the tip, I made the change.